ONSdigital/rm-sample-service
# SQL rebase of objects and standing data

# What and why?

Understanding if a rebase of the sample schema can be done by way of rebased liquibase sql scripts

# How to test?

Drop your existing schema and slash deploy this PR

`DROP SCHEMA sample CASCADE`

test acceptance tests run successfully following creation of rebased schema 

test idempotent redeploys of the app

no consideration currently for deploying into a persistent environment. These scripts do not currently check for the existence of objects before attempting to create

# Trello

https://trello.com/c/v3TpvNdq/1789-spike-rebase-sample-postgres-schema-5-days
